test/distributed/nn/jit/test_instantiator.py

```diff
@@ -33,7 +33,7 @@ def test_get_arg_return_types_from_interface(self):
             return_type_str,
         ) = instantiator.get_arg_return_types_from_interface(MyModuleInterface)
         self.assertEqual(args_str, "tensor, number, word")
-        self.assertEqual(arg_types_str, "tensor: Tensor, number: int, word: str")
+        self.assertEqual(arg_types_str, "tensor: Tensor, number: int, word: str = \"default\"")
```

Comment:
I think this is some test that checks that a schema is parsed in a specific way and the test was written in a way that didn't include defaults (because they weren't being parsed).